javascript/javascript1/week3/smartPhone-app.js

```diff
@@ -0,0 +1,91 @@
+// first part
+let activities = []
+function addActivity (date, activity, duration){
+
+    if (typeof date === 'string' && typeof activity === 'string' && typeof duration === 'number'){
+        
+        let object = {date, activity, duration}
+        activities.push(object)
+    }
+    
+}
+
+addActivity("23/7-18", "Youtube", 30);
+addActivity("23/7-18", "Youtube", 30);
+addActivity("23/7-18", "Youtube", 3);
+addActivity(2, '', 'I');
+addActivity("30/7-18", "facebook", 20);
+addActivity("1/3-21", "instagram", 15);
+addActivity("1/3-21", "instagram", 40);
+addActivity("1/3-21", "instagram", 50);
+console.log(activities)
+
+
+/*
+activities should now look like this
+[{
+    date: '23/7-18',
+    activity: 'Youtube',
+    duration: 30,
+}]
+*/
+
+// second part
+
+function showStatus (activities, timeLimit){
+    if (activities.length !== 0){
+        
+        let totalTime = 0;
+            for (let i = 0; i < activities.length; i++){
+                let time = activities[i].duration;
+                    totalTime += time
+            }
+                console.log(`You have added ${activities.length} activities. They amount to ${totalTime} min. of usage`)
+                    if (timeLimit <= totalTime){
+                         console.log(`You have reached your limit, no more smartphoning for you!`)    
+                    }   
```

Comment:
Your indentation is a bit off here.
```suggestion
        let totalTime = 0;
        for (let i = 0; i < activities.length; i++){
            let time = activities[i].duration;
            totalTime += time
        }
        console.log(`You have added ${activities.length} activities. They amount to ${totalTime} min. of usage`)
        if (timeLimit <= totalTime){
            console.log(`You have reached your limit, no more smartphoning for you!`)    
        }   
```